symplyphysics/laws/kinematic/vector/velocity_of_transfer_between_reference_frames.py

```diff
@@ -0,0 +1,64 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+    cross_cartesian_vectors,
+)
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The velocity related to such movement can be called transfer velocity.
+
+# Law: v_tr = v_0 + cross(w, r)
+## v_tr - vector of transfer velocity
+## v_0 - vector of velocity of moving origin O' of frame S' relative to fixed frame S
```

Comment:
What is moving origin?

Reply:
Every frame has an origin, and since S' is moving, O' is moving too. I don't really know what's unclear here.

Reply:
It is unclear, how this origin affects the result. As far as I understand, origin is any random point in S'. And every vector in S' should use this point as a starting point.
Add some notes to the description, if I am correct.

Reply:
Well, first of all, for any reference frame there is only _one_ chosen point of origin, so it's not _any random point_ technically. Moreover, not every vector in a reference frame has to start at that point. But these are just technical things. The main reason for that lies in the derivation of this law and I'm not really sure what note to write here. The choice of the origin affects the result in such a way that if we were to choose a different origin, we would have to re-calculate vectors and it might also move through space with a different velocity, but isn't that obvious?

Reply:
The problem with it, we have no mention of origin point before. That's why I want to clarify, what is it and how to choose one.
Suppose you are a user of this library and you have to pass origin velocity as parameter. How should you do it? What is origin of falling brick?
So I supposed, the choice of origin point is pretty much random. You choose one fixed point, that is the most convenient.
As a side note, there may be some problems in keeping this point fixed. For example, if the space of reference frame is bending, there is no obvious way of tracking absolute position of a given point.

Reply:
I'll mention the origin in the description. But isn't there only one origin for any reference frame? So defining a reference frame implies a definite origin.

The origin of falling brick is the origin of the coordinate system that we use to find the coordinates of the brick. It's not some arbitrary point in $`\mathbb{R}^3`$, otherwise we wouldn't have any coordinates.

And I think we only deal with flat spacetime within the scope of our library, so the space bending is out of question, although I can mention that.

Reply:
In law and description we should talk about velocity of a frame relative to another frame. And we should mention origin point as a way of finding this velocity, i.e. taking some fixed point in a frame and measuring its velocity.
Update description and naming according to this suggestion.